openmdao_book/other_useful_docs/om_command.ipynb

```diff
@@ -0,0 +1,626 @@
+{
```

Comment:
<p>There seems to be a mismatch between what's in the current docs for "openmdao n2" and this notebook version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Fixed</p>